File: src/Tawtheef.Websites/operations-web/src/app/pages/user/admin/offices-management/models/office.dto.ts
import {dropdownOptionsModel} from '../../../../../shared/models/dropdown-options.model';

export interface OfficeDto {
  id: string;
  nameAr: string;
  nameEn: string;
  country: dropdownOptionsModel;
  supportedCountries: dropdownOptionsModel[];
  adminEmail: string;
}

File: src/Application/Features/Operations/Admin/Offices/DTOs/OfficeDto.cs
using Tawtheef.Application.Common.Models;

namespace Tawtheef.Application.Features.Operations.Admin.Offices.DTOs;

public sealed record OfficeDto
{
    public Guid Id { get; init; }
    public string NameAr { get; init; } = string.Empty;
    public string NameEn { get; init; } = string.Empty;
    public required DropdownOptions Country {get; set;
    }
    public List<DropdownOptions> SupportedCountries { get; init; } = [];
    public string AdminEmail { get; set; } = string.Empty;
}
